wal_e/retries.py

```diff
@@ -70,6 +71,8 @@ def shim(*args, **kwargs):
                         exc_processor_cxt = exception_processor(
                             exception_info_tuple,
                             exc_processor_cxt=exc_processor_cxt)
+                    except UserCritical:
```

Comment:
I mostly have used ``UserCrticial`` as a shortcut. 
Note that only ``UserCritical`` exceptions raised by an ``exception_processor`` will be taken into account, and all of them are just logging errors.